sourcing/domain/service.go

```diff
@@ -0,0 +1,318 @@
+package domain
+
+import (
+	"context"
+	"math"
+
+	cartDomain "flamingo.me/flamingo-commerce/v3/cart/domain/cart"
+	"flamingo.me/flamingo-commerce/v3/cart/domain/decorator"
+	"flamingo.me/flamingo-commerce/v3/product/domain"
+
+	"flamingo.me/flamingo/v3/framework/flamingo"
+
+	"github.com/pkg/errors"
+)
+
+type (
+	// SourcingService interface
```

Comment:
perhaps add a few words here from the readme

Reply:
Hm, maybe it's a bit better now...